tests/runner.js

```diff
@@ -4,29 +4,31 @@
 var glob  = require('glob');
 var Mocha = require('mocha');
 var chalk = require('chalk');
-
+var root;
 
 require('traceur').require.makeDefault(function (filename) {
 // don't transpile our dependencies, just our app
-//The first check is if you develop locally, the second for the globally installed moduel
+// The first check is if you develop locally, the second for the globally installed moduel
   return (filename.indexOf('node_modules') === -1) ||
     (filename.indexOf('/node_modules/sane-cli/') > -1 && filename.indexOf('/node_modules/sane-cli/node_modules') === -1);
 }, {asyncFunctions: true});
 
 var mocha = new Mocha({
   // For some reason, tests take a long time on Windows (or at least AppVeyor)
-  timeout: (process.platform === 'win32') ? 90000 : 18000,
+  timeout : (process.platform === 'win32') ? 90000 : 18000,
   reporter: 'spec'
 });
 
 // Determine which tests to run based on argument passed to runner
 var arg = process.argv[2];
 if (!arg) {
-  var root = 'tests/{unit,acceptance,lint}';
+  root = 'tests/{unit,acceptance,lint}';
 } else if (arg === 'lint') {
-  var root = 'tests/lint';
+  root = 'tests/lint';
+} else if (arg === 'unit') {
```

Comment:
Added another option to only run unit tests, just as a convenience to avoid all the time it takes to run acceptance tests.